src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -239,6 +289,47 @@ export const NewProposal = ({ navigation, data, query }) => {
               </ScrollView>
             </>
           )}
+
+          {item.type === ITEM_TYPES.PICKER && (
+            <Controller
+              key={item.name}
```

Comment:
do we need a `key` here as we have no array that we loop for rendering several components?